notes/books/thinking-fast-slow.md

```diff
@@ -225,3 +306,50 @@ good and evil.
 
 It doesn't model reality, but our brains are not that interested in reality :-)
 (just the bare minimum to keep us alive and confortable).
+
+### Loss Aversion
+
+We are biased towards having more aversion to loss than to the benefits of gains.
+This comes from the fact that we pay more attention and learn faster from bad experiences.
+In nature it is more important to be able to identify predators and dangerous environments.
+You need to learn where you can gain things, like food, but knowing where danger is fundamental,
+you can be wrong multiple times about food and spend days without eating, you are going to be
+wrong about a predator only once.
+
+An interesting case of loss aversion is the behavior of most taxi drivers.
```

Comment:
I don't know if I understand well this use case. For me, it is easier to think about loss aversion with the following example:


In a financial investment scenario, an individual with loss aversion may be more concerned about avoiding a loss than maximizing potential gains. This aversion can lead to conservative choices, such as holding low-risk investments, even if there are opportunities for higher returns.

Reply:
This example is the one provided by the book that is part of a research (there is a full reference on the book for it). But yeah you will be able to see that in all sorts of human activities, it is very pervasive :D. I like the book example specially because it is subtle, you are doing something very irrational but in a very subtle way because of the bias.